### ISSUE TITLE
Accessibility issue: mini cart reduces opacity on hover

### ISSUE THREAD

**Issue by anphira** [woothemes/woocommerce#42328]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

For the mini cart gutenburg block, there is the following CSS:
.wc-block-mini-cart__button:hover:not([disabled]) {
    opacity: .6
}

This creates accessibility issues by violating WCAG 1.4.3.

### Expected behavior

I expect that colors I set for buttons & hovers, which I carefully check for contast, to not have opacities changed on me.

### Actual behavior

Opacity is reduced to only 60%. That means that very very few colors will actually pass contrast and be easily viewable. This is not normal or expected behavior to make an element harder to read on hover.

### Steps to reproduce

View source code of wc-blocks-style.css

### WordPress Environment

Not needed

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
